src/gc/gcpriv.h

```diff
@@ -2833,9 +2831,9 @@ class gc_heap
 
     PER_HEAP
 #ifndef MULTIPLE_HEAPS
-    CLREvent gc_done_event;
+    CLREvent* gc_done_event;
```

Comment:
This is `CLREvent*` pointer to `Impl*` pointer to actual stuff now. Would it make sense to get rid of at least one of these indirections at least - keep the Create method on GCEvent directly instead of having it on the GCToOS interface?